presto-main/src/main/java/com/facebook/presto/operator/TableFinishInfo.java

```diff
@@ -34,8 +32,8 @@
 {
     private static final int JSON_LENGTH_LIMIT = toIntExact(new DataSize(10, MEGABYTE).toBytes());
     private static final JsonCodec<Object> INFO_CODEC = jsonCodec(Object.class);
-    private static final JsonCodec<JsonNode> JSON_NODE_CODEC = jsonCodec(JsonNode.class);
 
+    private final Object connectorOutputMetadataInfo;
```

Comment:
What can this be?  Putting an arbitrary `Object` will be difficult to maintain/understand in the future.

Reply:
This can be List<String> . ConnectorOutputMetadata interface is currently only implemented by HiveWrittenPartitions and it returns List<String>